compliance/collection/main.go

```diff
@@ -283,7 +407,7 @@ func main() {
 	if features.RHCOSNodeScanning.Enabled() {
 		rescanInterval := env.NodeRescanInterval.DurationSetting()
 		cmetrics.ObserveRescanInterval(rescanInterval, getNode())
```

Comment:
[Not changed here, but] What is the point of reporting a metric for a value that does not change throughout the existence of the process? If we want to know the value, we can ask customers share their environment value overrides.

Reply:
Having the interval as metric would also enable us to see it e.g. in dashboards. That might be more useful for dev testing than operations, I agree.
It might be a better use to report the backoff time instead, as that one changes and can tell us a lot more about the state of Node Scanning.

Reply:
What would be the value of seeing the value on dashboards, what action may be taken based on the rescan interval duration, who is going to perform this action?
Sorry if it may feel offending, but I find it unsatisfactory to have the code _just because_ it's already there. There has to be a logic and reasoning, otherwise we end up with a system millions of subtle things in millions of subtle ways and so is a nightmare to validate and maintain.

Reply:
Okay, so the metric is provided as a baseline to inform users looking at Prometheus/Grafana about what is their actual node rescan interval. We provide it in order not to have to describe the rescan interval in the user documentation otherwise.

I still don't think the described scenario is convincing, but don't feel it's worth the fight to remove this metric.

Reply:
FYI, this metric is valuable when we start introducing randomized scan intervals: https://github.com/stackrox/stackrox/pull/5087